pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -69,6 +69,17 @@ const (
 	// ServiceAnnotationLoadBalancerResourceGroup is the annotation used on the service
 	// to specify the resource group of load balancer objects that are not in the same resource group as the cluster.
 	ServiceAnnotationLoadBalancerResourceGroup = "service.beta.kubernetes.io/azure-load-balancer-resource-group"
+
+	// ServiceAnnotationAllowedServiceTag is the annotation used on the service
+	// to specify a list of allowed service tags separated by comma
+	ServiceAnnotationAllowedServiceTag = "service.beta.kubernetes.io/azure-allowed-service-tags"
+)
+
+var (
+	// supportedServiceTags holds a list of supported service tags on Azure.
+	// Refer https://docs.microsoft.com/en-us/azure/virtual-network/security-overview#service-tags for more information.
+	supportedServiceTags = sets.NewString("VirtualNetwork", "VIRTUAL_NETWORK", "AzureLoadBalancer", "AZURE_LOADBALANCER",
+		"Internet", "INTERNET", "AzureTrafficManager", "Storage", "Sql")
```

Comment:
`AzureTrafficManager`, `Storage` and `Sql` are still not available in all azure clouds (e.g Germany). If the user would use one of those 3 serviceTag, where would they be provided the error message ?

Reply:
@djsly Augmented rules for NSGs has been GA: https://azure.microsoft.com/en-us/updates/agumented-rules-ga-nsg/

Reply:
Thanks, I guess since it is GA it is fine to keep them listed, I was jsut wondering what will happen on Clouds that do not yet support them. (e.g. Germany -- see below)
<img width="570" alt="screen shot 2018-03-22 at 07 43 30" src="https://user-images.githubusercontent.com/4981802/37768664-ce92df08-2da4-11e8-8c6c-5938e7ffb616.png">

Reply:
Azure ARM call will report errors on such case, and users could find the error message in service events (e.g. by kubectl describe service)

Reply:
thanks 👍